plugin/evm/atomic/txpool/mempool.go

```diff
@@ -224,16 +229,16 @@ func (m *Mempool) addTx(tx *atomic.Tx, local bool, force bool) error {
 	// We allow the transaction to be re-issued since it may have been invalid
 	// due to an atomic UTXO not being present yet.
 	if _, has := m.discardedTxs.Get(txID); has {
-		log.Debug("Adding recently discarded transaction %s back to the mempool", txID)
+		log.Debug("Adding recently discarded transaction back to the mempool",
+			"txID", txID,
+		)
```

Comment:
This is technically a fix, the logger doesn't format the string using `fmt`, it expects the arguments to be included as key-value pairs.